apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -50,8 +68,29 @@ export function AiGeneratePage({ resLocals }: { resLocals: Record<string, any> }
                 <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}" />
                 <input type="hidden" name="__action" value="generate_question" />
                 <div class="form-group">
-                  <label for="user-prompt-llm">Prompt:</label>
+                  <label for="user-prompt-llm"
+                    >Please give a high-level overview of the question. What internal parameters
+                    need to be generated and what information do we provide to students?</label
+                  >
```

Comment:
Formatting nit; please make the same change to the other labels below.

```suggestion
                  <label for="user-prompt-llm">
                    Please give a high-level overview of the question. What internal parameters
                    need to be generated and what information do we provide to students?
                  </label>
```

Reply:
Changed in 09ebd145316f730bdcbac5525887fda484b63483